rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/SendMessageDirectives.scala

```diff
@@ -35,6 +35,12 @@ trait SendMessageDirectives { this: ElasticMQDirectives with SQSLimitsModule =>
                   {messageAttributeDigest.map(d => <MD5OfMessageAttributes>{d}</MD5OfMessageAttributes>).getOrElse(())}
                   <MD5OfMessageBody>{digest}</MD5OfMessageBody>
                   <MessageId>{message.id.id}</MessageId>
+                  {message.messageAttributes.get("SequenceNumber").flatMap(sn => {
```

Comment:
shouldn't we use `SequenceNumberAttribute` here?

Reply:
:+1:

Reply:
ok, you're right :)